### DISCUSSION TITLE
How to translate enums to strings

### DISCUSSION THREAD

**Discussion by Loosemond** [facontidavide/PlotJuggler#761] [Q&A]
Hey

I'm reading my data from ros bags
Right now when I plot enums I only get the int value equivalent.
Is there a way to translate them or even make the translation while publishing so that I get the string name and still be able to plot the graph?

**Comment by facontidavide** [comment]
No, there isn't

**Reply by Loosemond** [reply]
Thanks,
Is it possible to create a plugin that does this? or implement directly in the code? (I can try to help) 
Ideal behavior would be like in python when you plot strings it treats it has states by default this way the user could translate enum to string and publish it. 
Don't know about performance or storage usage but it would make PlotJuggler usefull for more use cases.
